libraries/resource_abstractor_client/cluster_operations.py

```diff
@@ -0,0 +1,32 @@
+from requests import get, patch, put
+from resource_abstractor_client.client_helper import make_request
+
+RESOURCES_API = "/api/v1/resources"
+
+
+def get_resources(**kwargs):
+    return make_request(get, RESOURCES_API, params=kwargs) or []
```

Comment:
Let's say I am a user and I want to get the current resources that might be empty. I call this and under the hood, the make_requests fails for some reason - throws an exception and returns None, then this function returns []. 
Could this be a bit problematic for the user/debugging because you are not aware of an error occurring?
I hope you know what I mean.

Reply:
You are write I'll return None then.